angular_components/CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+## 0.10.1
+
+### Component Updates
+
+#### Material Chips
+* Improved support for generics.
```

Comment:
nit: Improved -> Improve (to be consistent with other entries).

Reply:
Thanks! I'll clean it up, not going to rerun this sync but it will be cleaner in the future.